templates/user/dashboard/feeds.tmpl

```diff
@@ -13,8 +13,8 @@
 						{{else if eq .GetOpType 2}}
 							{{$.i18n.Tr "action.rename_repo" .GetContent .GetRepoLink .ShortRepoPath | Str2html}}
 						{{else if eq .GetOpType 5}}
-							{{ $branchLink := .GetBranch | EscapePound}}
-							{{$.i18n.Tr "action.commit_repo" .GetRepoLink $branchLink .GetBranch .ShortRepoPath | Str2html}}
+							{{ $branchLink := .GetBranch | EscapePound | Escape}}
+							{{$.i18n.Tr "action.commit_repo" .GetRepoLink $branchLink (Escape .GetBranch) .ShortRepoPath | Str2html}}
```

Comment:
Why not use `$branchLink` twice instead of `(Escape .GetBranch)`?

Reply:
I think I chose to do that because EscapePound does url escaping and in general you would not want the branch names to be displayed like that (unless you want to punish users for using chars like `?#` and whitespace in their branch names :)

Reply:
makes sense.